docs/docs/dev/policies/timeout.md

```diff
@@ -88,3 +126,89 @@ Request timeouts are configured on the Envoy routes and may select a different T
 
 Mesh configures an idle timeout on the HTTPConnectionManager, but doesn’t consistently use the Timeout policy values for this, so the semantica are ambiguous.
 There’s no policy that configures the idle timeout for downstream connections to the Gateway.
+
+## Inbound timeouts
+
+Current policy configures timeouts only on the outbound side.
+Timeouts on the inbound side have constant values:
+
+```yaml
+connectTimeout: 10s 
+tcp:
+  idleTimeout: 2h
+http:
+  requestTimeout: 0s
+  idleTimeout: 2h
+  streamIdleTimeout: 1h
+  maxStreamDuration: 0s
+```
+
+These timeouts are either disabled or 2 times bigger than outbound default values
+to not interfere with the user's smaller values.
+
+Despite there is no policy at that moment to modify inbound timeouts,
+all values could be changed using ProxyTemplate:
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ProxyTemplate
+mesh: default
+metadata:
+  name: custom-template-1
+spec:
+  selectors:
+    - match:
+        kuma.io/service: '*'
+  conf:
+    imports:
+      - default-proxy 
+    modifications:
+      - networkFilter:
+          operation: patch
+          match:
+            name: envoy.filters.network.http_connection_manager
+            origin: inbound 
+          value: |
+            name: envoy.filters.network.http_connection_manager
+            typedConfig:
+              '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+              streamIdleTimeout: 0s # disable http.streamIdleTimeout 
+              common_http_protocol_options: 
+                idle_timeout: 0s # disable http.idleTimeout
+```
+:::
+
+::: tab "Universal"
+```yaml
+type: Timeout
+mesh: default
+name: timeout-all-default
+selectors:
+  - match:
+      kuma.io/service: '*'
+conf:
+  imports:
+    - default-proxy 
+  modifications:
+    - networkFilter:
+        operation: patch
+        match:
+          name: envoy.filters.network.http_connection_manager
+          origin: inbound 
+        value: |
+          name: envoy.filters.network.http_connection_manager
+          typedConfig:
+            '@type': type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+            streamIdleTimeout: 0s # disable http.streamIdleTimeout 
+            common_http_protocol_options: 
+              idle_timeout: 0s # disable http.idleTimeout
+```
+:::
+::::
```

Comment:
May here add a tip:

" You should always be careful and set the inbound timeout to be larger than the client's outbound timeouts. An easy rule of thumb is to set the inbound timeouts to be double the size of the client's outbound timeouts."